pkg/build/builder/dockerutil.go

```diff
@@ -49,6 +49,26 @@ type DockerClient interface {
 	TagImage(name string, opts docker.TagImageOptions) error
 }
 
+func pullImage(client DockerClient, name string, authConfig docker.AuthConfiguration) error {
+	logProgress := func(s string) {
+		glog.V(0).Infof("%s", s)
```

Comment:
@smarterclayton are you just suggesting it shouldn't be done at level 0?  I could agree w/ that potentially.